src/constants/externalResources.ts

```diff
@@ -123,7 +124,20 @@ export const getHoneycombSystemMetricsUrl = (
       { op: "AVG", column: "system.cpu.utilization" },
       { op: "RATE_AVG", column: "system.network.io.transmit" },
       { op: "RATE_AVG", column: "system.network.io.receive" },
-    ],
+    ].concat(
+      diskDevices
+        .map((device) => [
+          { op: "RATE_AVG", column: `system.disk.io.${device}.read` },
+          { op: "RATE_AVG", column: `system.disk.io.${device}.write` },
+          { op: "RATE_AVG", column: `system.disk.operations.${device}.read` },
+          {
+            op: "RATE_AVG",
+            column: `system.disk.operations.${device}.write`,
+          },
+          { op: "RATE_AVG", column: `system.disk.io_time.${device}` },
+        ])
+        .flat()
+    ),
```

Comment:
Disclaimer that the lint might be wrong so I wouldn't suggest committing this suggestion via the button 😅

```suggestion
      ...diskDevices
        .flatMap((device) => [
          { op: "RATE_AVG", column: `system.disk.io.${device}.read` },
          { op: "RATE_AVG", column: `system.disk.io.${device}.write` },
          { op: "RATE_AVG", column: `system.disk.operations.${device}.read` },
          {
            op: "RATE_AVG",
            column: `system.disk.operations.${device}.write`,
          },
          { op: "RATE_AVG", column: `system.disk.io_time.${device}` },
        ])
    ],
```

Reply:
nice! done